.github/workflows/nightly.yml

```diff
@@ -51,206 +51,66 @@ jobs:
-          - 6379:6379
-    env:
-      OS: ubuntu-latest
-      PYTHON: ${{ matrix.python-version }}
-      SQLALCHEMY_DATABASE_URI: ${{ matrix.db-uri }}
-      GITLAB_REMOTE_LOGIN_PAT: ${{ secrets.GITLAB_REMOTE_LOGIN_PAT }}
-      HOUSTON_DOTENV: ./.env
+
     steps:
-      # Checkout and env setup
       - uses: actions/checkout@v2
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python-version }}
-
-      - uses: actions/cache@v2
+        if: github.event_name == 'schedule'
         with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt', 'requirements/*.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
-
-      - name: Install libmagic on Ubuntu
-        run: |
-          sudo apt install libmagic1
-
-      - name: Install & Configure Houston
-        run: |
-          pip install "python-dotenv[cli]"
-          # Set secrets
-          dotenv -f ${HOUSTON_DOTENV} set GITLAB_REMOTE_LOGIN_PAT -- "${{ secrets.GITLAB_REMOTE_LOGIN_PAT }}"
-          dotenv -f ${HOUSTON_DOTENV} set SECRET_KEY -- "secret"
-          dotenv -f ${HOUSTON_DOTENV} set EDM_AUTHENTICATIONS_USERNAME__DEFAULT -- "${{ secrets.EDM_AUTHENTICATIONS_USERNAME__DEFAULT }}"
-          dotenv -f ${HOUSTON_DOTENV} set EDM_AUTHENTICATIONS_PASSWORD__DEFAULT -- "${{ secrets.EDM_AUTHENTICATIONS_PASSWORD__DEFAULT }}"
-          # Install application
-          ./scripts/install.sh
-
-      - name: Create frontend static directory (tests complains if static directory does not exist)
-        run: |
-          mkdir -p app/static/dist-latest
-          touch app/static/dist-latest/404.html
-          touch app/static/dist-latest/index.html
-          mkdir -p app/static/swagger-ui
-          touch app/static/swagger-ui/swagger-ui.js
-
-      - name: Check connectivity to required services
-        run: |
-          source virtualenv/houston3.7/bin/activate
-          invoke app.integrations.check
-
-      - name: Run tests (random x3)
-        run: |
-          source virtualenv/houston3.7/bin/activate
-          pytest -s -v --gitlab-remote-login-pat "${{ secrets.GITLAB_REMOTE_LOGIN_PAT }}"
-          pytest -s -v --gitlab-remote-login-pat "${{ secrets.GITLAB_REMOTE_LOGIN_PAT }}"
-          pytest -s -v --gitlab-remote-login-pat "${{ secrets.GITLAB_REMOTE_LOGIN_PAT }}"
-
-      - name: Check DB migrations (sqlite)
-        if: matrix.db-uri == 'sqlite://'
-        run: |
-          source virtualenv/houston3.7/bin/activate
-          # initialize the database
-          invoke app.db.upgrade --no-backup
-          # downgrade to the previous migration step
-          invoke app.db.downgrade
-          # upgrade to the latest version
-          invoke app.db.upgrade
-        env:
-          # Don't use in memory sqlite database for database migration
-          SQLALCHEMY_DATABASE_URI: ''
+          ref: main
```

Comment:
not 100% sure i understand the code, but for frontend we should be looking at develop branch!

Reply:
This is in reference to the `main` branch on the houston repo.